library/geonetwork

```diff
@@ -0,0 +1,28 @@
+# this file is generated via https://github.com/geonetwork/docker-geonetwork/blob/c3eba2643bc57c9679ad0cfce1ab5bd7e33fea2d/generate-stackbrew-library.sh
+
+Maintainers: Joana Simoes <joana.simoes@geocat.net> (@doublebyte1)
+GitRepo: https://github.com/geonetwork/docker-geonetwork
+
+Tags: 3.0.4
+GitCommit: b65e0df997508b59559e7f5e1be06ee12a8f8567
+Directory: 3.0.4
+
+Tags: 3.0.4-postgres
+GitCommit: 35a1391f8bd9c8d607cc9ac9fc1fa753160df79c
+Directory: 3.0.4/postgres
+
+Tags: 3.0.5, 3, latest
```

Comment:
Thanks for the comments @yosifkit :+1:  I implemented your suggestion of combining the SED instructions on one single line, on this [commit](https://github.com/geonetwork/docker-geonetwork/commit/e5cf5da76f557481c35b4a4d3e3cac77768a1302). The references are updated on [this](https://github.com/GeoCat/official-images/commit/96b9ccb845029612daea943ad3d10acc45a22198) commit.